rocketmq/client.py

```diff
@@ -397,15 +463,15 @@ def subscribe(self, topic, callback, expression='*'):
         def _on_message(consumer, msg):
             exc = None
             try:
-                callback(RecvMessage(msg))
-            except Exception as e:
+                return callback(RecvMessage(msg))
```

Comment:
Needs to check `callback` return type, should be `int` or `ConsumeStatus`

```python
return ConsumeStatus(callback(RecvMessage(msg)))
```

Will raise `ValueError` when callback's return type is not a valid `ConsumerStatus` variant.